$NetBSD: patch-multitail.conf,v 1.1 2025/05/27 19:15:12 vins Exp $

* Avoid hard-coding non portable utils path.
* Fix path to conversion scripts.

--- multitail.conf.orig	2024-08-13 10:49:48.000000000 +0000
+++ multitail.conf
@@ -1179,7 +1179,7 @@ cs_re:magenta:TLS
 #              end offset: the position AFTER the last character for which the attributes are
 #              valid
 #              do NOT use spaces in each line!
-colorscript:cscriptexample:/etc/multitail/colors-example.pl:this is a barely functional example script
+colorscript:cscriptexample:@PREFIX@/libexec/multitail/colors-example.pl:this is a barely functional example script
 #
 #
 # default colorschemes:
@@ -1231,7 +1231,7 @@ check_mail:0
 #
 # where to find the 'xclip' binary - used to send a buffer
 # to the X clipboard
-#xclip:/usr/bin/xclip
+xclip:@PREFIX@/bin/xclip
 #
 # where to find the 'pbcopy' binary - used to send a buffer
 # to the clipboard (OSX-only)
@@ -1244,8 +1244,8 @@ tab_stop:8
 # what program to start when the user presses a key
 # ...:x:...  just that key
 # ...:^x:... that key with control
-bind:g:/usr/bin/pine
-bind:^k:/bin/bash
+bind:g:/usr/bin/mail
+bind:^k:/bin/sh
 bind:^l:/usr/bin/telnet
 bind:^s:/bin/su
 #
@@ -1336,7 +1336,7 @@ convert:squid:ip4tohost:^[^ ]* *[0-9]* *
 convert:asterisk:epochtodate:^([0-9]*).([0-9]*)
 convert:nagios.log:epochtodate:^.([0-9]*). 
 convert:qmailtimestr:tai64todate:^(@[^ ]*)
-convert:geoip:script:/etc/multitail/convert-geoip.pl:([0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3})
+convert:geoip:script:@PREFIX@/libexec/multitail/convert-geoip.pl:([0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3})
 #
 # shell to invoke
 shell:/bin/sh
